### DISCUSSION TITLE
子图识别截取

### DISCUSSION THREAD

**Discussion by hsywu2020** [PaddlePaddle/PaddleOCR#12836] [Q&A]
#### 问题描述 / Problem Description
如果图片里含有子图，且没有直线边框分割，可以识别并截取吗？

#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:最新

#### 复现代码 / Reproduction Code
暂无

#### 完整报错 / Complete Error Message
无

**Comment by GreatV** [comment]
描述得太抽象了，可以给一个例子吗

**Reply by hsywu2020** [reply]
![ext_im3](https://github.com/PaddlePaddle/PaddleOCR/assets/122038849/525d60c9-9383-4525-b707-dda104919b78)
![ASPIRE_Dimopoulos_BCJ_2017_page3_figure1](https://github.com/PaddlePaddle/PaddleOCR/assets/122038849/c8b115e3-70a4-47d1-b90b-a573cf68943c)
例如这些，paddleocr只能识别整张大图，但是里面的子图上无法给框下来的，有什么办法吗

**Reply by hsywu2020** [reply]
图片是从整页的pdf里截取下来的，但是里面的子图，a，b，c，d 这些怎么能识别并截取呢？

**Reply by GreatV** [reply]
最符合要求的应该是版面分析功能了，但是当前版本不能返回图片中图的位置。

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/16b4f3dc-0107-4bb5-8f96-fe03831fc9ac)

**Comment by hsywu2020** [comment]
如果要子图的位置，有什么推荐的解决方案吗，因为子图的位置可能是任意的所以不太好办

**Reply by GreatV** [reply]
用目标检测的方法，把图标出来也行。

**Reply by hsywu2020** [reply]
目标检测模型里面有相应的代码吗？还是要自己写一个 非常感谢

**Reply by GreatV** [reply]
要自己训练和写代码
